src/olympia/devhub/views.py

```diff
@@ -1576,8 +1592,21 @@ def _submit_source(request, addon, version, submit_page, next_view):
             )
             VersionLog.objects.create(version_id=version.id, activity_log=activity_log)
             form.save()
+            log.info(
+                '_submit_source, form saved, addon.slug: %s, version.pk: %s',
+                addon.slug,
+                version.pk,
+            )
+            timer.log_interval('2.form saved')
```

Comment:
My suspicion is if there is any time spent between 1. and 2. it could be in either the validation (`is_valid`) or the saving (the rest) so maybe there should be timer straight after `is_valid`?

Reply:
Ok, I added logging for immediately after `is_valid`.